src/script/api/script_text.hpp

```diff
@@ -132,21 +132,21 @@ class ScriptText : public Text {
 	using Param = std::variant<SQInteger, std::string, ScriptTextRef>;
 
 	struct ParamCheck {
-		StringID owner;
+		uint owner;
 		int idx;
 		Param *param;
 		bool used = false;
 		const char *cmd = nullptr;
 
-		ParamCheck(StringID owner, int idx, Param *param) : owner(owner), idx(idx), param(param) {}
+		ParamCheck(uint owner, int idx, Param *param) : owner(owner), idx(idx), param(param) {}
 
 		void Encode(std::back_insert_iterator<std::string> &output, const char *cmd);
 	};
 
 	using ParamList = std::vector<ParamCheck>;
 	using ParamSpan = std::span<ParamCheck>;
 
-	StringID string;
+	uint string;
```

Comment:
I'd rather use a named type(def), so this is not erroneously converted back to `StringID` at some later point.

#13233 does that, though... do we want it to be a strong type or is `using StringIndexInTab = uint32_t` enough?

Reply:
Uh, I probably should've looked at your PR properly, sorry.